frontend/src/store/slices/UserSlice.ts

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { RootState } from "store/store";
+
+export interface User {
+    username: string;
+}
+
+const initialState: User = {
+    username: "Dohn Joe"
+};
+
+export const userSlice = createSlice({
+    name: "user",
```

Comment:
the global user state is the one nessesary global state

Reply:
do you mean that entire state is supposed to be placed in one global container? no slices whatsoever?

Reply:
I mean, that's one slice, which have sense, to be global, rest must be located locally